### ISSUE TITLE
Would you be interested in tailoring this guide for use with Forward Email's free plan?

### ISSUE THREAD

**Issue by titanism** [firstquadrant-ai/deliverabilityguide.com#3]
Hello @AnandChowdhary and hope you are doing well.

I'm sure you may have seen our service @forwardemail at https://forwardemail.net (since we were contributors to Upptime Monitor 😄, e.g. dark/light mode, and more; https://status.forwardemail.net).

Curious if you would be open to tailoring this guide to work with the free plan at https://forwardemail.net?

Recently we added the support of encrypted TXT records on the free plan, so that you can keep your forwarding configuration private from public DNS records.

See <https://forwardemail.net/en/faq> and the page to encrypt TXT records at <https://forwardemail.net/encrypt>.

> We allow you to encrypt records even on the free plan at no cost. Privacy should not be a feature, it should be inherently built-in to all aspects of a product. As highly requested in a [Privacy Guides discussion](https://discuss.privacyguides.net/t/forward-email-email-provider/13370) and on [our GitHub issues](https://github.com/forwardemail/forwardemail.net/issues/254) we've added this.

**Comment by AnandChowdhary** [comment]
Definitely interesting. How do you imaging this would work? Like adding a section in the docs with recommendations and a tutorial on how to set it up?

**Comment by securitybrahh** [comment]
Note you can't forward without bringing in your own domain, or you can get vanity domain at 3$/month from these guys iirc.

**Comment by titanism** [comment]
@securitybrahh Correct, you should bring your own domain name, but we do offer one of our own if you pay $3/mo.  It's cheaper to use us on the free plan with your own domain name (e.g. $3*12=$36, and most domains cost ~$5-10/yr &ndash; and it's encouraged vs. using ours actually).  You should own your presence on the Internet and not be tied to a single provider for life.

**Comment by securitybrahh** [comment]
> you should bring your own domain name

Any provider you like that is reliable + nokyc?  (Always looking for new such domain registrars 😋)

**Comment by titanism** [comment]
@securitybrahh use https://namecheap.com and `COUPONFCNC` every year for renewal

**Comment by titanism** [comment]
@securitybrahh also we offer coupon code `GITHUB` for 100% off one month of our paid plans

**Comment by AnandChowdhary** [comment]
Haha I'm glad this repository is an official support channel now haha ;)
